consoles/localXvnc.pm

```diff
@@ -27,11 +27,23 @@ require IPC::System::Simple;
 use Socket;
 use File::Which;
 
+# helper function
+# Keep ssh session for the maximum of ServerAliveCountMax x ServerAliveInterval seconds
+# even without receiving any messsage back from the server, and this will not affect normal
+# ssh disconnect and console switching. Ssh console may not display returned result of
+# long-time run test without these options. TCPKeepAlive ensures that if network goes down
+# or the remote host dies, machines will be properly noticed
 sub sshCommand {
     my ($self, $username, $host, $gui) = @_;
 
-    my $sshopts = "-o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no -o PubkeyAuthentication=no $username\@$host";
-    $sshopts = "-X $sshopts" if $gui;
+    my $server_alive_count_max = get_var('_SSH_SERVER_ALIVE_COUNT_MAX', 480);
```

Comment:
`get_var()` is not defined in this module. You need to add `use testapi;`. All our s390 workers are currently broken because of that.

Reply:
https://openqa.suse.de/tests/4723195